src/NuGet.Core/NuGet.Frameworks/NuGetFramework.cs

```diff
@@ -218,6 +294,25 @@ public virtual string GetShortFolderName(IFrameworkNameProvider mappings)
                             framework.DotNetFrameworkName));
                     }
                 }
+                else if (IsNet5Era)
```

Comment:
The code inside this block looks good, I'm just wandering if we are benefiting from the IsNet5Era check.